packages/permission_handler/README.md

```diff
@@ -57,4 +57,3 @@ You can use this plugin to ask the user for runtime permissions if your app perf
 ## Limitations
 
 - This plugin is only supported on **Galaxy Watch** devices running **Tizen 5.5** or later. On **TV**s, you don't need to request permissions since they are already granted to apps by default.
```

Comment:
```suggestion
## Notes

This plugin is intended for **Galaxy Watch** devices only. On **TV**s, you don't need to request permissions since they are already granted to apps by default.
```

Now supports Tizen 4.0.

Reply:
I've update it, thanks :)